packages/shared/lib/core/market/stores/market-coin-prices.store.ts

```diff
@@ -0,0 +1,14 @@
+import { writable } from 'svelte/store'
+import { MarketCoinPrices } from '../types'
+
+export const marketCoinPrices = writable<MarketCoinPrices>(undefined)
+
+export function updateMarketCoinPrices(payload: Partial<MarketCoinPrices>): void {
+    return marketCoinPrices.update((state) => {
+        if (marketCoinPrices) {
```

Comment:
is this correct? `marketCoinPrices` is never undefined because it is a writable. Do you mean:
 ```suggestion
        if (payload) {
```